### ISSUE TITLE
Make help modals accessible

### ISSUE THREAD

**Issue by abias** [moodleuulm/moodle-theme_boost_campus#117]
If you enable the `theme_boost_campus | helptextmodal` setting, the help popovers from Moodle core are changed to modal windows.

Unfortunately, these modals can be accessed by using a keyboard only (and pressing the `tab` button). This accessibility flaw should be fixed.

**Comment by abias** [comment]
As we have documented on https://moodle.org/plugins/theme_boost_campus, Boost Campus will not be continued after Moodle 3.x anymore as we will focus on the successor Boost Union which is published on https://moodle.org/plugins/theme_boost_union. For the time being, as long as Moodle 3.x still receives security support, we will only fix critical bugs in Boost Campus if one arises. And, of course, we try to port over all relevant features of Boost Campus and open proposals for Boost Campus to Boost Union.

Against this background, this feature and this issue has been ported over to Boost Union on https://github.com/moodle-an-hochschulen/moodle-theme_boost_union/issues/123

Cheers,
Alex
